### ISSUE TITLE
Yolov8 Pose Landmark Detection, not getting full success

### ISSUE THREAD

**Issue by cafermutluozkan** [ultralytics/ultralytics#18155]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello,
I detect points on the face in Cephalometric Images.
I used Yolov8x-pose and Yolo11x-pose models for this.
Training was done over 1000 images.
Train= 800
Valid= 100
Test= 100

I created the labels myself.
I made the labels according to the following order.
Example Label 1
`0 0.683254 0.603351 0.492632 0.599330 0.811962 0.592268 2 0.819617 0.577320 2 0.499522 0.564433 2 0.843541 0.727320 2 ...`
and
`kpt_shape: [44, 3]`

Example Label 2
`0 0.683254 0.603351 0.492632 0.599330 0.811962 0.592268 0.819617 0.577320 0.499522 0.564433 0.843541 0.727320... `
`kpt_shape: [44, 3] or kpt_shape: [44, 2]`

Here, when I make Labels like “Example Label 1”
Yolov8 model found 43 points out of 44 points as I wanted.
The yolo11 model found 43 out of 44 points as I wanted.
In both tutorials I find a different point too outlier!

Here when I make the labels like “Example Label 2”
On Yolov8 he found almost none of them in the right places.
Yolo11 training was not done.

Here I would like to ask the following questions;
1-) Why didn't I succeed in all points in “Sample Tag 1”?
2-) In “Sample Tag 2” I deleted the “2” in the txt files, what is the importance of this? Is this wrong?
3-) What is the importance of “2” or “3” in the kpt_shape parameter?

**Comment by UltralyticsAssistant** [comment]
👋 Merhaba @cafermutluozkan, Ultralytics'e olan ilginiz için teşekkürler 🚀! Soru ve detaylı açıklamalarınız için de ayrıca teşekkür ederiz. İşte size yardımcı olabilecek birkaç öneri ve bilgi:

1. **Belgelere Göz Atın** 🌟: Yeni başlayanlar için [Dokümantasyon](https://docs.ultralytics.com)’da [Python](https://docs.ultralytics.com/usage/python/) ve [CLI](https://docs.ultralytics.com/usage/cli/) kullanım örnekleri yer almakta. Ek olarak, [Model Eğitim İpuçları](https://docs.ultralytics.com/guides/model-training-tips/) sayfasını inceleyerek eğitiminizin kalitesini artıracak yöntemler keşfedebilirsiniz.

2. **Minimum Çoğaltılabilir Örnek (MRE)** 🐛: Eğer bu bir **hata raporu** ise, lütfen sorunu yeniden oluşturabilmemiz için [minimum çoğaltılabilir bir örnek (MRE)](https://docs.ultralytics.com/help/minimum_reproducible_example/) paylaşın. Bu, eğitim sırasında kullandığınız kod parçacıkları, parametreler ya da küçük bir veri örneğini içerebilir.

3. **Özel Sorular❓**: Eğer bu bir **özel eğitimle alakalı bir soru** ise, lütfen daha fazla bilgi paylaşın:
   - Kullanılan hyperparameter ayarları
   - Dataset’tan örnek görseller
   - Eğitim ve validasyon logları
   - Etiket format açıklaması ve örnekleri

4. **Topluluk ve Destek** 🎧: Ultralytics topluluğu size yardımcı olmak için burada! Gerçek zamanlı sohbet için [Discord](https://discord.com/invite/ultralytics)'a katılabilir, uzun tartışmalar için [Discourse](https://community.ultralytics.com)’u kullanabilir ya da bilgi paylaşımı için [Subreddit](https://reddit.com/r/Ultralytics)’te sorularınızı sorabilirsiniz.

---

## Güncelleme

En son sürümdeki olası hata düzeltmelerinden faydalanmak için lütfen Python ortamınızda aşağıdaki komutu çalıştırarak `ultralytics` paketini güncelleyin:

```bash
pip install -U ultralytics
```

Ayrıca, [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) dosyasındaki bağımlılıkların uygun olduğundan emin olun (örneğin, [Python>=3.8](https://www.python.org/) ve [PyTorch>=1.8](https://pytorch.org/get-started/locally/)).

---

## Çalışma Ortamları

Aşağıdaki doğrulanmış ortamları kullanarak YOLO modellerini çalıştırabilirsiniz (her biri CUDA/CUDNN ve diğer bağımlılıkları içerir):
- **Notebooks**: Ücretsiz GPU ortamıyla çalıştırın:  
    <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>  
    <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>  
    <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
    
- **Google Cloud** Deep Learning VM. Daha fazla bilgi için [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).  
- **Amazon** Deep Learning AMI. Daha fazla bilgi için [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).  
- **Docker Image**. Detaylar için [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/):  
  <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>  

---

## Durum

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>  
Bu rozetin yeşil olması, tüm [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) testlerinin başarıyla geçtiği anlamına gelir. Bu testler, [Modlar](https://docs.ultralytics.com/modes/) ve [Görevler](https://docs.ultralytics.com/tasks/) için her 24 saatte bir tüm işletim sistemlerinde (macOS, Windows, Ubuntu) çalıştırılır.

---

Bu otomatik bir yanıttır 🤖. Bir Ultralytics mühendisi sorularınızı inceleyecek ve yakında size yardımcı olacaktır.

**Comment by Y-T-G** [comment]
1. You probably need a lot more images. 44 points is like having 44 classes. 800 images are not enough for that.
2. It indicates visibility. https://github.com/cocodataset/cocoapi/issues/130#issuecomment-369147997
3. If you don't have have visibility label, it would be 2 values per keypoint (x and y). If you have visibility label, you will have 3 values (x, y and visibility).

**Comment by cafermutluozkan** [comment]
@Y-T-G  Thank you for your valuable answers.
